commands/audit/audit.go

```diff
@@ -160,6 +161,10 @@ func (auditCmd *AuditCommand) CommandName() string {
 	return "generic_audit"
 }
 
+func (auditCmd *AuditCommand) HasViolationContext() bool {
+	return len(auditCmd.watches) > 0 || auditCmd.projectKey != "" || auditCmd.targetRepoPath != ""
```

Comment:
why targetRepoPath is here?

Reply:
if you provide repoPath they can infer the project as far as I know and violations returns